src/profile/ChangePassword.js

```diff
@@ -29,39 +27,42 @@ const ChangePassword = (props) => {
   };
 
   const handleChange = (event) => {
-    setstate({
+    setState({
       ...state,
-      [event.target.id]: event.target.value
+      [event.target.id]: event.target.value,
     });
   };
 
   const handleChangeClick = async (event) => {
     event.preventDefault();
-    setstate({
+    setState({
       ...state,
-      isChanging: true
+      isChanging: true,
     });
 
     try {
       const currentUser = await Auth.currentAuthenticatedUser();
-      await Auth.changePassword(currentUser,state.oldPassword,state.password);
+      await Auth.changePassword(
+        currentUser, 
+        state.oldPassword, 
+        state.password
+      );
```

Comment:
After the Auth function returns, before you run the history object to move to a new route, can you add a `successToast` that says 'Password successfully changed.' Can be imported from the same place as the errorToast. Thank you!

Reply:
If you search for `errorToast` or `successToast` throughout the repository, you will see examples of this in action!

Reply:
Ok @mikhael28 . I will work on the required changes 🙂